tests/test_handlers.py

```diff
@@ -581,12 +581,12 @@ def test_am1_differences():
             assert 0
 
 
-def test_compute_or_load_am1_charges():
+def test_compute_or_load_am1_charges_and_matches():
```

Comment:
Nit: naming -- test renamed but definition doesn't seem to reflect this change

Reply:
Fixed in [21039c3](https://github.com/proteneer/timemachine/pull/640/commits/21039c3556e0ba96419309922e6b5c41f8411330)